README.md

```diff
@@ -32,35 +32,40 @@ Cron is an application which will execute commands on a defined schedule. WebFac
 
 All of the commands listed below (unless specified otherwise) are run in an SSH session on the server. Again, WebFaction has written a [splendid little tutorial on how to get this working](https://docs.webfaction.com/user-guide/access.html#ssh).
 
+### SFTP
+
+If you're not happy navigating around your server's folders and files through SSH, you might find some of this process easier if you access your server with an FTP client over Secure FTP. WebFaction [also has this covered](https://docs.webfaction.com/user-guide/access.html#connecting-with-ftp).
+
```

Comment:
I like this! The user will have to use SSH to set up the Cron job at some point, but if the bulk of the task is made easier by SFTP, that's great.